src/libs/OptionsListUtils.ts

```diff
@@ -1661,7 +1668,7 @@ function getUserToInviteOption({
         showChatPreviewLine,
     });
     userToInvite.isOptimisticAccount = true;
-    userToInvite.login = searchValue;
+    userToInvite.login = isValidEmail || isValidPhoneNumber ? searchValue : '';
```

Comment:
Make sure it doesn't break filtering logic for participants screen and others (by looking at the usage)